node/pkg/governor/mainnet_chains.go

```diff
@@ -10,21 +10,21 @@ import (
 
 func chainList() []chainConfigEntry {
 	return []chainConfigEntry{
-		chainConfigEntry{emitterChainID: vaa.ChainIDSolana, dailyLimit: 50000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDEthereum, dailyLimit: 50000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDTerra, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDBSC, dailyLimit: 5000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDPolygon, dailyLimit: 5000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAvalanche, dailyLimit: 5000000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDOasis, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAlgorand, dailyLimit: 200000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAurora, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDFantom, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDKarura, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAcala, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDKlaytn, dailyLimit: 1_000_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDCelo, dailyLimit: 500000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDNear, dailyLimit: 200000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDTerra2, dailyLimit: 500000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDSolana, dailyLimit: 50_000_000, bigTransactionSize: 5_000_000},
+		chainConfigEntry{emitterChainID: vaa.ChainIDEthereum, dailyLimit: 50_000_000, bigTransactionSize: 5_000_000},
```

Comment:
Wondering if with the addition of auto-release functionality whether the `dailyLimit` could be drawn down on Solana/Ethereum if big tx's ($5M+ in this context) will always have finality delay.  If taking the last 30-days of Solana/Ethereum tx's (less the $5M+ txs) would this still result in a 50M recommended config for `dailyLimit` those chains or could it be lower?

Reply:
I'd think it's also worth considering this analysis for the other chains, it would just seem Ethereum/Solana would seem to have the most impactful risk reductions if they could be set lower by many M after discounting the big tx's.